examples/rpc/partner-plugin/services/data/v1/shared_data.go

```diff
@@ -0,0 +1,76 @@
+package helpers
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"path/filepath"
+	"sync"
+
+	accommodationv1 "buf.build/gen/go/chain4travel/camino-messenger-protocol/protocolbuffers/go/cmp/services/accommodation/v1"
+)
+
+var (
+	properties []accommodationv1.PropertyExtendedInfo
+	loadOnce   sync.Once
+)
+
+func LoadPropertiesMockData() []accommodationv1.PropertyExtendedInfo {
+	loadOnce.Do(func() {
+		// Assuming the JSON file is relative to the current file
+		currentDir, err := os.Getwd()
+		if err != nil {
+			log.Printf("Error getting current directory: %v", err)
+			return
+		}
+
+		filePath := filepath.Join(currentDir, "../../examples", "rpc", "partner-plugin", "mock_data", "properties.json")
```

Comment:
Is it best-practice to build up paths like this in go? This looks a bit weird to me.

Asked Evgenii and he mentioned that it's also possible to embed json files directly at compile time like this:
```go
//go:embed ../../examples/rpc/partner-plugin/mock_data/properties.json
var propertiesJSON []byte
```

Reply:
loading of the json file has been refactored in the proposed way